### ISSUE TITLE
Migration to organization

### ISSUE THREAD

**Issue by Neustradamus** [arpcard/card-live-dashboard#8]
It is possible to migrate the personal account to organization?

Converting:
- https://help.github.com/articles/converting-a-user-into-an-organization/

It is possible to rename this account and create the organization and move/transfer the repository to the organization.

Rename account:
- https://help.github.com/en/enterprise/2.13/user/articles/changing-your-github-username

Create an organization:
- https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch

Transfer a repository:
- https://help.github.com/en/github/administering-a-repository/transferring-a-repository

Like:
- http://github.com/gnome
- http://github.com/kde
- ...

**Comment by Neustradamus** [comment]
Dear @arpcard team,

My initial ticket is very old (2020-11-06), can you realize this request?

Thanks in advance.

**Comment by agmcarthur** [comment]
Apologies, we will look into this shortly.

**Comment by agmcarthur** [comment]
@raphenya let's work on this after the May 2025 release.

**Comment by Neustradamus** [comment]
@agmcarthur: Thanks for your answer :)

Before or after, the result is same, why not to do before?
There is no lost, etc.
The conversion is easy and only need to select the team members (minimal one good personal account).

**Comment by agmcarthur** [comment]
Hi @Neustradamus, timing is just based on staff availability.

**Comment by Neustradamus** [comment]
@agmcarthur: Yes, I understand, no problem but before to do the May 2025 release is possible, it takes few seconds :)

**Comment by agmcarthur** [comment]
@raphenya is working on this now!

**Comment by agmcarthur** [comment]
Done.

**Comment by Neustradamus** [comment]
@agmcarthur: Good job, thanks!

**Comment by Neustradamus** [comment]
@agmcarthur, @raphenya: Can you add you in Public (instead of Private) here:
- https://github.com/orgs/arpcard/people

PS: Please inform team members too.
